src/sql/parts/dashboard/widgets/insights/views/charts/chartInsight.component.ts

```diff
@@ -19,6 +19,8 @@ import { IColorTheme, IWorkbenchThemeService } from 'vs/workbench/services/theme
 import * as nls from 'vs/nls';
 import { ITelemetryService } from 'vs/platform/telemetry/common/telemetry';
 
+declare var Chart: any;
```

Comment:
I think the general recommendation is to use `let` instead of `var` in typescript.